test/resources/models/inference_example.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+import json
+import logging
+import re
+from collections import namedtuple
+
+import requests
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+Context = namedtuple('Context',
+                     'model_name, model_version, method, rest_uri, grpc_uri, '
+                     'custom_attributes, request_content_type, accept_header')
+
+DEFAULT_TFS_MODEL = 'half_plus_three'
+DEFAULT_TFS_METHOD = 'predict'
+DEFAULT_CONTENT_TYPE = 'application/json'
+DEFAULT_ACCEPT_HEADER = 'application/json'
+BASE_URI = 'http://localhost:8080/invocations'
+
+JSON_CONTENT_TYPE = ['application/json', 'application/jsons', 'application/jsonlines']
+
+
+def input_handler(data, context):
+    """ Pre-process request input before it is sent to TensorFlow Serving REST API
+
+    Args:
+        data (obj): the request data, in format of dict or string
+        context (Context): an object containing request and configuration details
+
+    Returns:
+        (dict): a JSON-serializable dict that contains request body and headers
+    """
+
+    if context.request_content_type == 'application/json':
```

Comment:
is the example meant to be an example for users or code for our tests? the multiple input types may make the example look more complicated than it needs to.